src/cgame/cg_weapons.cpp

```diff
@@ -1182,10 +1182,15 @@ static void CG_CalculateWeaponPosition( vec3_t out_origin, vec3_t out_angles )
 	VectorCopy( cg.refdefViewAngles, offsets.angles );
 	VectorClear( offsets.angvel );
 
-	if( !filter.IsEmpty( ) )
+	auto last = filter.Last( );
```

Comment:
This would be an out-of-bounds access the first time the function is called. I made #2678 to fix this function